tests/test_validation.py

```diff
@@ -57,7 +66,12 @@ def test_warnings_reana_yaml(
     warnings = validate_reana_yaml(reana_yaml)
     assert set(expected_warnings.keys()) == set(warnings.keys())
     for key, value in expected_warnings.items():
-        assert set(value) == set(warnings[key])
+        if isinstance(value, list):
+            assert len(value) == len(warnings[key])
+            for warning_value in value:
+                assert warning_value in warnings[key]
+        else:
+            assert operator.eq(value, warnings[key])
```

Comment:
Just as a curiosity, why we call `operator.eq` here and not `==`?

Reply:
Relic of a previous version of the test in which I used `map` and passed `operator.eq` as a function. I've changed it to `==` to be more pythonic :wink: